website/tests/testcli.sh

```diff
@@ -0,0 +1 @@
+<snippet id='shell-install-command'/>
```

Comment:
This runs in CI already (the build failed for other reasons. this part passed.)

Reply:
What is this testing?

Reply:
in this particular case, that whatever we are using to install shell works. This would be followed by actual shell usage when this goes out of the WIP stage (e.g. invoke near state test.near as the next line)

Reply:
An immediate thought is we're going to have commands like those here:
https://docs.nearprotocol.com/docs/local-setup/local-dev-testnet#4-create-an-account-for-your-contract
`near create_account <yourcontractname> --masterAccount=<login> --initialBalance <initalbalance>`
in multiple places that would benefit from being snippets, but shouldn't be tested. wonder if we should add an attribute `notest` or something.
But this comment is not a blocking one. I think this is great.

Reply:
Tests are pretty much optional right now (i.e. someone needs to explicitly make a test for something to be tested), so what you are suggesting pretty much is the default situation anyways.